config.ts

```diff
@@ -6,3 +6,33 @@ export const defaultConfig: Config = {
   maxPagesToCrawl: 50,
   outputFileName: "output.json",
 };
+
+
+// const treeEndPointUrl = 'https://github.com/BuilderIO/gpt-crawler/tree/main'
+// const blobEndPointUrl = 'https://github.com/BuilderIO/gpt-crawler/blob/main'
+
+// export const defaultConfig: Config = {
+//   url: "https://github.com/BuilderIO/gpt-crawler/tree/main",
+//   match: [
+//     {
+//       // skip the pattern you do not want to crawl
+//       // pattern: "https://github.com/BuilderIO/gpt-crawler/tree/main/**",
+//       pattern: `${treeEndPointUrl}/**`,
+//       skip: true
+//     },
+//     {
+//       // speical case for .md
+//       // for .md, we need to crawl the raw content in the .markdown-body selector
+//       // pattern: 'https://github.com/BuilderIO/gpt-crawler/blob/main/**/*.md',
+//       pattern: `${blobEndPointUrl}/**/*.md`,
+//       selector: '.markdown-body'
+//     },
+//     {
+//       // other files like .js, .ts, .json, etc
+//       pattern: `${blobEndPointUrl}/**`,
+//       selector: '#read-only-cursor-text-area'
+//     },
+//   ],
+//   maxPagesToCrawl: 50,
+//   outputFileName: "output.json",
+// };
```

Comment:
This is the test code for the new `match` parameter option because there is no test process for the project. Should I delete it or just put this example to README?

Reply:
Removed